storage/migrations/02_runtimes.up.sql

```diff
@@ -207,6 +207,19 @@ CREATE TABLE chain.evm_token_balance_analysis
 );
 CREATE INDEX ix_evm_token_balance_analysis_stale ON chain.evm_token_balance_analysis (runtime, token_address, account_address) WHERE last_download_round IS NULL OR last_mutate_round > last_download_round;
 
+CREATE TABLE chain.evm_contracts
+(
+  runtime runtime NOT NULL,
+  contract_address oasis_addr NOT NULL,
+  creation_tx HEX64 NOT NULL,
+  creation_bytecode BYTEA NOT NULL,
+  runtime_bytecode BYTEA,
+  verified BOOLEAN NOT NULL,
+  -- Fields below are only populated for verifed contracts.
+  abi JSONB,
+  PRIMARY KEY (runtime, contract_address)
```

Comment:
Nit: I've been pushing for a convention where the `PRIMARY KEY` and `FOREIGN KEY` lines come right after the primary/foreign keys. See (most) other tables for an example; can you move this one?

It's more readable (IMO) and visually closer to single-row PKs like `my_id INTEGER PRIMARY KEY`

Reply:
Updated! Also standardized the other tables as well.